src/health_misinfo_shared/fine_tuning.py

```diff
@@ -324,7 +324,7 @@ def get_video_responses(
                     json_text = candidate.text
                     json_text = tidy_response(json_text)
```

Comment:
I'm not sure if `tidy_response` is strictly needed with the new parsing method

Reply:
I tried removing it and got an error, but it's hard to trace if that's because I removed the line, or something else.

Reply:
Going to keep it in for safety I think

Reply:
`tidy_reponse` and `parse_model_json_output` are two different ways of attempting to correct badly-formatted JSON. So I think keeping both is good - but I'm not sure why they're separate functions in the first place? 
But it looks fine for now.

Reply:
I've now moved `tidy_reponse` into `parse_model_json_output`. I only had crashed when I completely removed `tidy_response` and that could just be chance but I've kept it in just to be safe, just hidden away.